rust/src/lib/unit_tests/bond.rs

```diff
@@ -781,3 +781,32 @@ fn test_bond_opt_ns_ip6_target() {
         assert_eq!(e.kind(), ErrorKind::InvalidArgument);
     }
 }
+
+#[test]
+fn test_bond_ports_ports_config_conflict() {
```

Comment:
Why we have test case for a clean up patch?

The test case should belong to bug fix patch or standalone commit.